packages/cli/src/runIOS/runIOS.js

```diff
@@ -28,7 +28,7 @@ const getBuildPath = function(configuration = 'Debug', appName, isDevice) {
     device = 'iphonesimulator';
   }
 
-  return `build/Build/Products/${configuration}-${device}/${appName}.app`;
+  return `build/${scheme}/Build/Products/${configuration}-${device}/${appName}.app`;
```

Comment:
this will fail with undefined scheme, right? can we guard against it?

Reply:
Defined here: https://github.com/react-native-community/react-native-cli/blob/7d7238a7d53b42fe3b92517a0f89b1f038a16f28/packages/cli/src/runIOS/runIOS.js#L69